test-cases/manager/manager-multiple-restores-schema-and-data.yaml

```diff
@@ -0,0 +1,19 @@
+test_duration: 2100
+
+round_robin: true
+
+instance_type_db: 'i3en.3xlarge'
```

Comment:
Not on all runs as far as I know. It was even Roy that recommended that instance type in the first place

Reply:
This instance type is needed only for the large scale dataset (5TB).
What is the dataset size you test here? 2TB?
If so, it can be reduced to significantly, especially if you use 5 db nodes.
(calculate exactly what you need).

Reply:
The dataset will fit on i3en,2xlarge, yes, but the run time is increased significally as a result - a restore that previously took 13.5 hours now takes over 18.5. I'm not sure it's worth it.

Reply:
I kept the instance type at 3xlarge, I don't believe the extra run time is worth it

Reply:
Leave it for now.
Please share somwhere (in the test plan probably) the runtimes we know.

Reply:
Added to the test plan